app/pods/cases/case/subcases/subcase/overview/route.js

```diff
@@ -9,7 +9,16 @@ export default class CasesCaseSubcasesSubcaseOverviewRoute extends Route {
 
   async afterModel(model) {
     // For showing the history of subcases within this route, we need a list of subcases without the current model
-    const allSubcases = this.modelFor('cases.case.subcases');
+    //  We want to sort descending on date the subcase was concluded.
+    //  In practice, reverse sorting on created will be close
+    const allSubcases = await this.store.query('subcase', {
+      filter: {
+        case: {
+          id: this.case.id,
+        },
+      },
+      sort: '-created',
+    });
```

Comment:
I get that the pagination was not compatible with the current setup where a parent model from route `cases/case/subcase` is passed down to this afterModel.
Querying these here will impact the performance of the subcase overview?
use-case:
Case with 10 subcases, I want to view all of the info
before PR: loaded all subcases once when loading the parent route, opening detail view 10 times by going back to the overview and opening another subcase just passes on that model.
after PR: We query subcases on the overview route, we open the first subcase and we need to query all subcases again. We go back to the overview and have to query all subcases, opening second subcase is another query to all subcases. and so on.

Am I seeing this wrong?
If I'm not, perhaps it would be a good idea to keep the call in the parent route to avoid doing it multiple times.
The queries will be cached after the first time, but cache invalidations happen often on active cases/subcases.

Reply:
The problem with keeping it in the parent route is that we force the `subcase` rout to only show whatever page we're on in the `subcases` route. Additionally, in a follow-up story we add pagination to the `subcase` route too, which should be separate from the pagination in the `subcases` route.

I don't know what the real life use case is like though, if there will be tons of `cases` with >25 `subcases`, so I don't know how often pagination is actually going to be used here.

Reply:
I'll keep the possible performance hit in mind going forward.
Now that I know this is mainly a legacy data issue I guess querying the same thing a few times will be fine in terms of performance on average cases.